### PR TITLE
Update STaaS.md - Backups

### PR DESCRIPTION
**Pull request by joaobarraca**

### CODE REVIEW

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 1]
```diff
@@ -77,6 +77,14 @@ In addition, the following **other limitations** currently apply:
 
 - DOM queries can be slower depending on the number of DOM records and the complexity of the query. This limitation will be removed in the near future.
 
+## Backups
+
+Your data is backed up **daily** and stored for **30 days**.
+
+- Daily Backups: STaaS performs daily backups, maintaining a 30-day rolling snapshot of your data. This procedure ensures a comprehensive data history for potential restorations.
```
Comment (RobbertVH) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:84]: Maybe emphasize that this is against user mistakes/accidental changes. That this is about data resilience, not data redundancy.

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 2]
```diff
@@ -77,6 +77,14 @@ In addition, the following **other limitations** currently apply:
...
+
+- Data Restoration and Support: In the event a roll-back is necessary, our support team is prepared to assist. Please submit a rollback request by opening a ticket with our support team. They will guide you through the necessary steps to ensure a successful data restoration.
```
Comment (RobbertVH) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:86]: are we sure that will be the eventual flow ?

File: user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md [hunk 3]
```diff
@@ -65,6 +65,14 @@ DataMiner STaaS relies on Azure Storage, which stores multiple copies of your da
 > [!TIP]
 > For detailed information, see [Azure Storage redundancy on learn.microsoft.com](https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy)
 
+## Data resilience and backups
+
+To ensure data resilience for potential recovery scenarios, protecting against user errors and accidental changes, your data is backed up with a **granularity of 1 day**. Backups are stored for **30 days**.
+
+- **Daily Backups**: STaaS performs backups with a granularity of 1 day and maintains a 30-day rolling snapshot of your data.
+
+- **Data Restoration and Support**: In the event a rollback is necessary, our support team will assist you. Submit a rollback request by opening a ticket with the support team. They will guide you through the necessary steps to ensure a successful data restoration.
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md:74]: To make this more user-friendly, could you add a link to the email address of the support team here?

Comment (joaobarraca) [reply user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md]: done

**Pull request merged**